examples/xlora_inference/xlora_inference_mistralrs.py

```diff
@@ -0,0 +1,37 @@
+"""
+Perform inference of an X-LoRA model using the inference engine mistral.rs
```

Comment:
Let's put this info into a README.md in the same directory instead of the docstring. Maybe also worth mentioning that this supports other architectures than mistral, as the name might suggest otherwise.

Reply:
Added that!